core/net/src/hybrid/service.rs

```diff
@@ -474,6 +477,10 @@ fn forward_bus_to_net(
     let state = state.clone();
     let request_id = gen_id().to_string();
 
+    packet_trace_maybe!("net::forward_bus_to_net", {
+        try_extract_from_ip_frame(msg)
+    });
+
```

Comment:
I would use qualified importshere like so:
```rust
ya_packet_trace::packet_trace_maybe!("net::forward_bus_to_net", {
    try_extract_from_ip_frame(msg)
});
```
It would enable to drop:
```rust
#[allow(unused_imports)]
use ya_packet_trace::{packet_trace_maybe, try_extract_from_ip_frame};
```
in every file

Reply:
Done